src/Symfony/Component/Validator/DependencyInjection/AddValidatorInitializersPass.php

```diff
@@ -49,11 +49,19 @@ public function process(ContainerBuilder $container)
 
         // @deprecated logic, to be removed in Symfony 5.0
         $builder = $container->getDefinition($this->builderService);
-        $calls = [];
+        $calls = array();
 
         foreach ($builder->getMethodCalls() as list($method, $arguments)) {
             if ('setTranslator' === $method) {
-                $translator = $arguments[0] instanceof Reference ? $container->findDefinition($arguments[0]) : $arguments[0];
+                if ($arguments[0] instanceof Reference) {
```

Comment:
Just setting the reference as "nullable" wouldn't work, we'd have the same error. Though, your put is interesting: I've made sure it ignores the reference **only** if it's "nullable".

Reply:
Actually no. I've just reverted 3e545e4e3e1874a88a293fdaff107d0bad561af8 as it would break the `ValidatorBuilderInterface`. AFAIK, that's the only way of dealing with it here.